build/Microsoft.DotNet.Cli.Compile.targets

```diff
@@ -121,36 +129,26 @@
              File="%(SdkOutputChModTargets.FullPath)"
              Mode="%(SdkOutputChModTargets.Mode)" />
 
-      <DotNetPublish ToolPath="%(Stage.DotnetDir)"
-                     Output="$(SdkOutputDirectory)"
-                     Framework="netcoreapp1.0"
-                     ProjectPath="$(SrcDirectory)/compilers" />
-
-      <Copy SourceFiles="$(BinaryToCorehostifyOutDir)/csc.exe"
-            DestinationFiles="$(SdkOutputDirectory)/csc.dll" />
+      <RemoveAssetFromDepsPackages DepsFile="$(SdkOutputDirectory)/redist.deps.json"
+                                   SectionName="runtimeTargets"
+                                   AssetPath="$(BinaryToCorehostifyRelDir)/%(BinariesToRemove.Identity).exe" />
 
-      <Move SourceFiles="$(BinaryToCorehostifyOutDir)/csc.exe"
-            DestinationFiles="$(SdkOutputDirectory)/csc.exe" />
+      <Copy SourceFiles="$(SdkOutputDirectory)/%(FilesToMove.Identity)"
+            DestinationFiles="$(SdkOutputDirectory)/%(FilesToMove.Filename).dll" />
 
-      <Copy SourceFiles="$(SdkOutputDirectory)/compilers.deps.json"
-            DestinationFiles="$(SdkOutputDirectory)/csc.deps.json" />
+      <Copy SourceFiles="$(SdkOutputDirectory)/redist.deps.json"
+            DestinationFiles="$(SdkOutputDirectory)/%(BundledTools.Identity).deps.json" />
 
-      <Copy SourceFiles="$(SdkOutputDirectory)/compilers.runtimeconfig.json"
-            DestinationFiles="$(SdkOutputDirectory)/csc.runtimeconfig.json" />
+      <Copy SourceFiles="$(SdkOutputDirectory)/redist.runtimeconfig.json"
+            DestinationFiles="$(SdkOutputDirectory)/%(BundledTools.Identity).runtimeconfig.json" />
 
       <ChangeEntryPointLibraryName
-            DepsFile="$(SdkOutputDirectory)/csc.deps.json"
-            NewName="csc.deps.json" />
+            DepsFile="$(SdkOutputDirectory)/%(BundledTools.Identity).deps.json"
+            NewName="%(BundledTools.Identity).deps.json" />
 
-      <RemoveAssetFromDepsPackages DepsFile="$(SdkOutputDirectory)/csc.deps.json"
-                                   SectionName="runtimeTargets"
-                                   AssetPath="$(BinaryToCorehostifyRelDir)/%(BinariesToRemove.Identity).exe" />
-      <RemoveAssetFromDepsPackages DepsFile="$(SdkOutputDirectory)/dotnet.deps.json"
-                                   SectionName="runtimeTargets"
-                                   AssetPath="$(BinaryToCorehostifyRelDir)/%(BinariesToRemove.Identity).exe" />
-
-      <!-- cleanup compilers project output we don't need -->
-      <Delete Files="$(SdkOutputDirectory)/compilers%(PublishOutputExtensions.Identity)" />
```

Comment:
Do we still need to delete the compilers output?  How about the src\nuget output?